### PR TITLE
Added :google command.

### PR DESCRIPTION
**Pull request by 0xMoJo7**
Fixes issue #4

### CODE REVIEW

**Comment by rahuldecoded** [comment]
Where is change in the config file? `:google <user_query>`

**Comment by 0xMoJo7** [comment]
ah sry, doing now.

**Comment by rahuldecoded** [comment]
while commiting use `git commit --amend --no-edit`

**Comment by 0xMoJo7** [comment]
apologies, i am rather new to git

File: google.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    url         = 'http://www.google.com/search'
+    payload     = { 'q' : search_string }
+    my_headers  = { 'User-agent' : 'Mozilla/11.0' }
+    r           = requests.get( url, params = payload, headers = my_headers )
+    soup        = BeautifulSoup( r.text, 'html.parser' )
+    h3tags      = soup.find_all( 'h3', class_='r' )
+    for h3 in h3tags:
+        try:
+            temp.append( re.search('url\?q=(.+?)\&sa', h3.a['href']).group(1) )
+        except:
```
Comment (rahuldecoded) [inline google.py]: Don't leave except `blank`

File: google.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            continue
+    temp = temp[:3]
+    return temp
```
Comment (rahuldecoded) [inline google.py]: Here `temp` is list. We need to return a string.

**Comment by rahuldecoded** [comment]
Good job till now. Do the required changes.

**Comment by 0xMoJo7** [comment]
is that a good solution here?

**Comment by rahuldecoded** [comment]
@emdavis7 Do the necessary changes.

File: google.py (hunk 1 above)
Comment (0xMoJo7) [reply google.py]: this change or something else?

**Review by rahuldecoded — COMMENTED** [review]
Replace `except` with `except Exception`

File: google.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    r           = requests.get( url, params = payload, headers = my_headers )
+    soup        = BeautifulSoup( r.text, 'html.parser' )
+    h3tags      = soup.find_all( 'h3', class_='r' )
+    for h3 in h3tags:
+        try:
+            temp.append( re.search('url\?q=(.+?)\&sa', h3.a['href']).group(1) )
+        except:
+            return "Something went wrong with your search query."
+    temp = temp[:3]
+    return temp
```
Comment (rahuldecoded) [inline google.py]: replace `temp` with `', '.join(temp)`

**Comment by rahuldecoded** [comment]
Thanks 😀

**Comment by 0xMoJo7** [comment]
no, thank you for the lesson :) again, appreciate the patience

**Comment by rahuldecoded** [comment]
@emdavis7 feel free to ping me @ 2806rahul@gmail.com
And ping us at gitter link is in README

**Pull request merged**